lib/util/trip/trip_card.dart

```diff
@@ -180,9 +182,7 @@ abstract class TripCardState<T extends Trip, U extends TripCard<T>> extends Stat
     return const SizedBox();
   }
 
-  Widget buildRightSide() {
-    return const SizedBox();
-  }
+  Widget buildRightSide();
```

Comment:
wait why did you change this? some other card might not want to implement this

Reply:
Because it's overwritten in every subclass but i can change it Back.

Reply:
Yeah but that's not guaranteed right?

Reply:
I think the change is fine. It serves as a reminder that TripCard is, indeed, abstract, and if an extending class doesn't want anything there, it should implement that itself. This just forces us to think about it

Reply:
No? It's much better if you just fill the spaces that you want. The rest should obviously always be empty boxes, so that should also be the default given in `TripCard`.

Reply:
I see two different philosophies here. I like it explicit, you like it implicit. That's fine, but I like it explicit

Reply:
okay so what do i do now? should i change it back or not? what do the others think @Showcas @MarkFalke

Reply:
In any case, you need to do something. Either you remove all of these "build<Position>" methods and add the stubs to the descendants. Or you just remove this change and implement it in this class again.

Reply:
ok i guess i just change it back then xD